File: hw/yanez/u3/HW21TemplateMethod/template_method_python/template_method/controller/beverage_test.py
from template_method.model.tea import Tea
from template_method.model.coffee import Coffee


def main():
    tea = Tea()
    coffee = Coffee()
    print("\nMaking tea ..")
    tea.prepare_recipe()
    print("\nMaking coffee ...")
    coffee.prepare_recipe()

if __name__ == "__main__":
    main()

File: hw/bonilla/U3/HW21TemplateMethod/TemplateMethod/src/main/java/ec/edu/espe/templatemethod/controller/BeverageTest.java
package ec.edu.espe.templatemethod.controller;

import ec.edu.espe.templatemethod.model.Coffee;
import ec.edu.espe.templatemethod.model.Tea;

/**
 *
 * @author Bonilla David SoftCrafters
 */
public class BeverageTest {

    public static void main(String[] args) {
        Tea tea = new Tea();
        Coffee coffee = new Coffee();
        System.out.println("\nMaking tea .."); 
        tea.prepareRecipe();
        System.out.println("\nMaking coffee ...");
        coffee.prepareRecipe();
    }
}
